packages/shared/locales/en.json

```diff
@@ -577,15 +577,18 @@
             },
             "proposalsDetails": {
                 "title": "Details",
+                "totalProposals": "Total proposals",
                 "activeProposals": "Active proposals",
                 "votingProposals": "Proposals you are voting on",
                 "votedProposals": "Total proposals you voted on"
             },
             "proposals": {
                 "title": "Proposals",
                 "voted": "Voted",
-                "successRegister": "Proposal successfully registered",
-                "successRemove": "Proposal successfully removed"
+                "successAdd": "Proposal successfully added",
+                "successRemove": "Proposal successfully removed",
+                "emptyTitle": "No proposals",
+                "emptyDescription": "Add a proposal now!"
```

Comment:
Don't kill me if I don't :rofl:

Reply:
I'll ask design to get something smoother here :D

Reply:
😂 Just adapted from collectibles

Reply:
We can remove it, since it's the same message as the button.

Reply:
Removed the "emptyDescription"